src/renderer/pages/Governance/model/governance-page-model.ts

```diff
@@ -32,10 +34,37 @@ sample({
 //   target: referendumDetailsModel.events.referendumChanged,
 // });
 
+sample({
+  clock: [referendumFilterModel.events.queryChanged, governanceModel.$ongoingReferendums],
+  source: {
+    referendums: governanceModel.$ongoingReferendums,
+    details: referendumListModel.$referendumsDetails,
+    chain: networkSelectorModel.$governanceChain,
+    query: referendumFilterModel.$query,
+  },
+  filter: ({ chain }) => Boolean(chain),
+  fn: ({ referendums, details, chain, query }) =>
+    filterReferendums({ referendums, details, query, chainId: chain!.chainId }),
```

Comment:
I think better to add `{}` because we start return from the new line, same for next sample